arm-datalake-analytics/catalog/2016-11-01/swagger/catalog.json

```diff
@@ -2767,11 +3343,12 @@
           "description": "the assembly file type.",
           "enum": [
             "Assembly",
-            "Resource"
+            "Resource",
+            "Nodeploy"
           ],
           "x-ms-enum": {
             "name": "FileType",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
Thanks for pointing this out and including it. Sorry, I missed that in the top comment. Sounds good.